RecoHGCal/TICL/plugins/HGCGraph.cc

```diff
@@ -54,29 +54,29 @@ void HGCGraph::makeAndConnectDoublets(const TICLLayerTiles &histo,
         auto const &outerLayerHisto = histo[currentOuterLayerId];
         auto const &innerLayerHisto = histo[currentInnerLayerId];
 
-        for (int oeta = startEtaBin; oeta < endEtaBin; ++oeta) {
-          auto offset = oeta * nPhiBins;
-          for (int ophi_it = startPhiBin; ophi_it < endPhiBin; ++ophi_it) {
-            int ophi = ((ophi_it % nPhiBins + nPhiBins) % nPhiBins);
-            for (auto outerClusterId : outerLayerHisto[offset + ophi]) {
+        for (int ieta = startEtaBin; ieta < endEtaBin; ++ieta) {
```

Comment:
Is this inversion of the order of looping through innerLayer and outerLayer due to some performance improvement expected?

Reply:
Ciao @perrotta 
this change will orient the search cone from inner to outer, rather than the opposite (which was wrong).